### PR TITLE
Increase test Coverage of pkg/templates

### PR DESCRIPTION
**Pull request by raja-sami-10p**
Signed-off-by: Raja Sami <raja.sami@tenpearls.com>

**- What I did**
Increase unit test coverage of pkg/templates

**- How to verify it**
```sh
make test-unit
```

**- A picture of a cute animal (not mandatory but encouraged)**
:horse:

### CODE REVIEW

File: pkg/templates/templates_test.go [hunk 1]
```diff
@@ -66,5 +70,17 @@ func TestParseTruncateFunction(t *testing.T) {
 		var b bytes.Buffer
 		assert.NoError(t, tm.Execute(&b, source))
 		assert.Equal(t, testCase.expected, b.String())
+
+		t.Run("Empty Source Test with template: "+testCase.template, func(t *testing.T) {
```
Comment (cpuguy83) [inline pkg/templates/templates_test.go]: Can we either convert the test above this to use the `t.Run` pattern or continue to use the already established pattern in this test?

**Review by cpuguy83 — CHANGES_REQUESTED** [review]

File: pkg/templates/templates_test.go (hunk 1 above)
Comment (raja-sami-10p) [inline pkg/templates/templates_test.go]: I've changed it to t.Run pattern.

**Review by cpuguy83 — APPROVED** [review]
LGTM

**Comment by thaJeztah** [comment]
windows failing on https://github.com/moby/moby/issues/33805, which is a known flaky test

**Review by thaJeztah — APPROVED** [review]
LGTM

**Pull request merged**